release/models/qos/openconfig-qos-elements.yang

```diff
@@ -35,7 +35,13 @@ submodule openconfig-qos-elements {
       packets for transmission, including policer and shaper
       functions";
 
-  oc-ext:openconfig-version "0.11.2";
+  oc-ext:openconfig-version "0.11.3";
+
+  revision "2025-05-22" {
+    description
+      "Add two new enums for scheduler-policy priority.";
+    reference "0.11.3";
```

Comment:
Following https://github.com/openconfig/public/blob/master/doc/semver.md#general-guidelines-for-versioning-openconfig-yang-modules, should this be `0.12.0`?